### PR TITLE
fix bug 1155504 - add crash-analysis host

### PR DESCRIPTION
**Pull request by rhelmer**
r? @phrawzty @jdotpz - I think this is the most straightforward config right now for an EC2 instance running just Nginx.

The puppet bit seems pretty reasonable, but we really need to cut down on the amount of terraform config needed :)

### CODE REVIEW

File: terraform/analysis/main.tf [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    description = "Allow (alt) SSH to the CrashAnalysispp node."
+    ingress {
+        from_port = "${var.alt_ssh_port}"
+        to_port = "${var.alt_ssh_port}"
+        protocol = "tcp"
+        cidr_blocks = [
+            "0.0.0.0/0"
+        ]
+    }
+    tags {
```
Comment (phrawzty) [inline terraform/analysis/main.tf]: Should the tags here (and elsewhere) also have the `role` and `project` tags (like in #57)?

**Comment by phrawzty** [comment]
What does this role do? The service config is very, very light - could be anything?

**Comment by phrawzty** [comment]
> The puppet bit seems pretty reasonable, but we really need to cut down on the amount of terraform config needed :)

Yeah - I'd really like to make use of templates more (I guess?).

**Comment by rhelmer** [comment]
> What does this role do? The service config is very, very light - could be anything?

The bug has a little more info - crash-analysis just hosts some Apache UserDirs, I expect it'll need a few other things to run server-side reports (PHP at least, I could at that now I guess - note that it only needs the PHP CLI command, nginx doesn't need to run PHP)

**Comment by rhelmer** [comment]
@phrawzty OK I think that addresses your comments, lmk

File: terraform/analysis/main.tf [hunk 2]
```diff
@@ -0,0 +1,131 @@
+provider "aws" {
+    region = "${var.region}"
+    access_key = "${var.access_key}"
+    secret_key = "${var.secret_key}"
+}
+
+resource "aws_security_group" "any_to_analysis__ssh" {
+    name = "${var.environment}__any_to_analysis__ssh"
+    description = "Allow (alt) SSH to the CrashAnalysispp node."
```
Comment (phrawzty) [inline terraform/analysis/main.tf]: Speeling. :grin:

**Comment by phrawzty** [comment]
Squash and then `r+` w/ minor nits.

**Pull request merged**